src/main/java/MaxMinAverage.java

```diff
@@ -0,0 +1,59 @@
+/*
+1.1. Максимальное, минимальное и среднее значение
+Задача:
+Заполните массив случайным числами и выведете максимальное, минимальное и среднее значение.
+Для генерации случайного числа используйте метод Math.random().
+*/
+
+
+import java.util.ArrayList;
+
+public class MaxMinAverage {
+
+    public static ArrayList<Integer> generateRandomArray(int arrayLength) {
+        ArrayList<Integer> newArray = new ArrayList<Integer>();
```

Comment:
You may omit "Integer" on the right side

Reply:
Thanks for the comment. This was fixed.